data/map.txt

```diff
@@ -17790,7 +17790,7 @@ planet Dune
 planet Dustmaker
 	attributes hai
 	landscape land/badlands0
-	description `The surface of this planet, when it is not obscured by dust storms, receives enough sunlight to power not only large solar farms, but solar furnaces as well, which the Hai use for manufacturing a large variety of goods. Some of the solar farms are so large that from orbit, the light reflecting off them could almost be mistaken for lakes and small oceans. However, there is almost no water on the planet's surface except near the poles.`
+	description `The surface of this planet, when it is not obscured by dust storms, receives enough sunlight to power not only large solar farms, but solar furnaces as well, which the Hai use for manufacturing a large variety of goods. Some of the solar farms are so large that from orbit the light reflecting off them could be mistaken for lakes and small oceans. However, there is almost no water on the planet's surface except near the poles.`
```

Comment:
Maybe.  My grammar isn't always textbook perfect, but I don't often see prepositional phrases separated by commas.  Maybe the relative clause 'that from orbit' might be separated by double commas, I'm not sure.  If it had to keep a comma somewhere, placing a single comma after 'large' seems to make more sense to me.

I removed the comma here because that is what the person who made the original PR did.

Could you point me to the comma rule(s) you used to make that correction please?